dbt_coves/tasks/setup/dbt.py

```diff
@@ -0,0 +1,98 @@
+import os
+from pathlib import Path
+
+import questionary
+from jinja2 import Environment, meta
+from rich.console import Console
+
+from dbt_coves.utils.jinja import render_template
+from dbt_coves.utils.shell import run, run_and_capture_cwd
+from .utils import print_row
+
+
+console = Console()
+
+
+class SetupDbtTask:
```

Comment:
This task should have `register_parser` and `run` methods, so it can be called as `dbt-coves setup dbt`